misaligned.js

```diff
@@ -1,16 +1,15 @@
-const {expect} = require('chai')
+function index_to_number(i,j){
+    return i * 5 + j;
+}
 
 function print_color_map() {
     const majorColors = ["White", "Red", "Black", "Yellow", "Violet"];
-    const minorColors = ["Blue", "Orange", "Green", "Brown", "Slate"];
+    const minorColors = ["Blue", "Orange", "Green", "Brown","Slate"];
     for (let i = 0; i < majorColors.length; i++) {
         for (let j = 0; j < minorColors.length; j++) {
-            console.log(`${i * 5 + j} | ${majorColors[i]} | ${minorColors[j]}`);
+            console.log(`${index_to_number(i,j)} | ${majorColors[i]} | ${minorColors[j]}`);

```

Comment:
This statement makes it hard to test the alignment. Separate the forming of the string into another function as well, then test the output of that function to see if all the columns are aligned.